### PR TITLE
Margins

### PR DESCRIPTION
**Pull request by jwong-d2l**

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -15,16 +15,17 @@
     "wct.conf.json"
   ],
   "dependencies": {
-    "polymer": "Polymer/polymer#^2.0.0",
```
Comment (jwong-d2l) [inline bower.json]: alphabetical order

File: d2l-sequence-viewer.html [hunk 2]
```diff
@@ -41,7 +41,7 @@
 					flex-flow: row;
 				}
 				#sidebar-occlude {
-					width: calc(50% - 615px - 2px);
```
Comment (jwong-d2l) [inline d2l-sequence-viewer.html]: new breakpoints as per nick

**Comment by jbydeley-d2l** [comment]
These new breakpoints don't work very well. The top header is still `1230px` breakpoint and the sidebar now has a pixel range where it's not fully visible.

image

File: d2l-sequence-viewer.html [hunk 3]
```diff
@@ -20,6 +20,9 @@
 		<custom-style include="d2l-typography">
 			<style is="custom-style" include="d2l-typography">
 				:host {
+					--viewer-max-width: 1170px;
+					--sidebar-position: calc(50% - var(--viewer-max-width) / 2);
```
Comment (jbydeley-d2l) [inline d2l-sequence-viewer.html]: Since these values are related, might as well make them understandable.

Comment (jwong-d2l) [reply d2l-sequence-viewer.html]: genius!

**Pull request merged**